pkg/cmd/pr/status/status.go

```diff
@@ -179,20 +179,20 @@ func prSelectorForCurrentBranch(baseRepo ghrepo.Interface, prHeadRef string, rem
 	return
 }
 
-func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
+func printPrs(cs *iostreams.ColorScheme, w io.Writer, totalCount int, prs ...api.PullRequest) {
 	for _, pr := range prs {
 		prNumber := fmt.Sprintf("#%d", pr.Number)
 
-		prStateColorFunc := utils.Green
+		prStateColorFunc := cs.Green
 		if pr.IsDraft {
-			prStateColorFunc = utils.Gray
+			prStateColorFunc = cs.Gray
 		} else if pr.State == "MERGED" {
-			prStateColorFunc = utils.Magenta
+			prStateColorFunc = cs.Magenta
 		} else if pr.State == "CLOSED" {
-			prStateColorFunc = utils.Red
+			prStateColorFunc = cs.Red
 		}
```

Comment:
This does not need to be cleaned up in the PR but it seems like this is just a duplicate of `ColorForPR`.